src/api/TestCaseServiceApi.test.tsx

```diff
@@ -173,7 +173,7 @@ describe("TestCaseServiceApi Tests", () => {
     const readTestCaseCb = jest.fn();
     testCaseService.readTestCaseFile(file, readTestCaseCb);
     await waitFor(() => {
-      expect(readTestCaseCb).toHaveBeenCalledWith(
+      expect(readTestCaseCb).not.toHaveBeenCalledWith(
```

Comment:
I think this is a test case for checking invalid files. any reason to change expectation to `not` ?

Reply:
Because of the new check, it doesn't hit this any more. Should the test be removed instead?

Reply:
Neither. It should still hit. The check sets the `errorMessage`, which gets passed to the callback function and is then displayed to the user as a toast.